src/renderers/dom/server/ReactServerRenderingTransaction.js

```diff
@@ -7,12 +7,14 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactServerRenderingTransaction
+ * @flow
```

Comment:
Ok, I'll type those files completely then, but by experience, that's how flow works, it will not pass if for instance, ReactServerRenderingTransaction is not annotated...